src/test/java/com/google/api/codegen/testdata/discoveries/csharp/csharp_dataproc.v1.json.baseline

```diff
@@ -38,108 +38,6 @@ namespace DataprocSample
                 ApplicationName = "Google Cloud Platform Sample",
             });
 
```

Comment:
So we're still generating the same baseline files, but the contents just happened to change? OK, I guess. Though weird. It seems like a breaking change to the API surface....